Decred Wallet.xcodeproj/project.pbxproj

```diff
@@ -1072,7 +1070,6 @@
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(inherited)",
 					"$(PROJECT_DIR)/libs",
-					"$(PROJECT_DIR)",
```

Comment:
This is not complete. There are other build configs that still have this value set.